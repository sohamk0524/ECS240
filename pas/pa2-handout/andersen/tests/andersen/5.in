c Test case with address-of and single deref on LHS and RHS
p 4 5
s 0 1 0 4
s 1 1 1 2
s 0 2 -1 3
s 0 3 -1 4
s 0 4 -1 1
