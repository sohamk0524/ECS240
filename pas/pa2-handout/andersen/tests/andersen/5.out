pt 1 1
pt 1 4
pt 2 3
pt 3 4
pt 4 1
pt 4 4
